.github/workflows/test-swift-sdk.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Swift SDK
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  test:
+    name: Integration Tests
+    runs-on: macos-latest
+    steps:
+      - name: Checkout Sources
+        uses: actions/checkout@v4
+
+      - name: Install Flipt
+        uses: flipt-io/setup-action@v0.1.1
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.75.0
+          override: true
+
+      - name: Install Swift
+        uses: swift-actions/setup-swift@v1
+        with:
+          swift-version: "5.9"
+
+      - name: run flipt
+        env:
+          FLIPT_STORAGE_TYPE: "local"
+          FLIPT_STORAGE_LOCAL_PATH: "./test/fixtures/testdata"
+        run: flipt&
+
+      - name: Install Rust Targets
+        run: |
+          rustup target add aarch64-apple-ios x86_64-apple-ios aarch64-apple-ios-sim
+
+      - name: Build Engine
```

Comment:
this duplicates some work by recompiling the binaries and packaging the xcframework here, but i figured the small time tradeoff is worth the simpler workflow. and this completes before the rest of the integration tests anyway